img/draw_fov.pro

```diff
@@ -20,7 +20,8 @@ PRO		DRAW_FOV, DET=DET, ALL=ALL, GOOD=GOOD, TARGET=TARGET, XYCOR=XYCOR, YEAR=YEA
 ;
 
 
-	default, year, 2014
+default, year, 2014
+COMMON PARAM
```

Comment:
this seems like a dangerously vanilla name for a common block. Could it be changed to FOXSI_PARAM or something?